threatminer/komand_threatminer/util/utils.py

```diff
@@ -0,0 +1,85 @@
+from komand.exceptions import PluginException
+
+import json
+from typing import Dict, Iterable, Any
+import requests
+
+
+def get_query(url: str, query: str) -> requests.Response:
+    try:
+        response = requests.get(url, params={"q": query}, allow_redirects=False)
+    except requests.exceptions.HTTPError as e:
+        raise PluginException(
+            cause="Unexpected http error",
+            data=str(e),
+            assistance="If the issue persists please contact support.",
+        )
+    check_status_code(
```

Comment:
You might be able to replace this with 
`response.raise_for_status()` 

Then handle the exception as needed.